biojava-modfinder/src/main/java/org/biojava/nbio/phosphosite/Site.java

```diff
@@ -56,25 +56,25 @@ public Site(){
 
 		int proteinIndex = -1;
 		int uniprotIndex = -1;
-		int modTypeIndex = -1;
 		int residueIndex = -1;
 		int orgIndex     = -1;
 		int groupIndex   = -1;
+		int geneIndex    = -1;
 
 		boolean inHeader = true;
 
 
 		while ((line = buf.readLine()) != null){
-			if ( line.startsWith("PROTEIN")) {
+			if ( line.startsWith("GENE")) {
```

Comment:
The other change was that they dropped a column (modification type) in the latest version. I think it can be inferred from the annotation in the "MOD_RSD" column (which the latest version of the parser is doing).